DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -161,8 +165,11 @@ function Set-TargetResource
     if ($IsBlue)
     {
         Write-Verbose "Set values into the web.config that define the repository for BLUE OS"
-        PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbprovider" -value $eseprovider
-        PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbconnectionstr"-value $esedatabase
+        #PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbprovider" -value $eseprovider
```

Comment:
See https://github.com/PowerShell/Whitepapers/blob/master/PullServerCPIG.docx advanced configuration section. Michael Greene reconfigures to MDB as EDB is missing. It is also my experience that EDB is never there so installation is always in error. I changed these lines to correct this issue at configuration time.
![missingedb](https://cloud.githubusercontent.com/assets/10071039/7331937/29eb22fc-eb27-11e4-9377-393edbc4b7d7.png)